fineract-provider/src/main/resources/static/api-docs/apiLive.htm

```diff
@@ -9438,170 +9585,1409 @@ <h4>Retrieve Group accounts overview</h4>
+
+						<dt>sortBy</dt>
+						<dd>
+							String <span>optional</span>, one of <span>ASC, DESC</span>
+						</dd>
+						<dd>Indicates what way to order results if <i>orderBy</i> is used.</dd>
+
+						<dt>officeId</dt>
+						<dd>
+							Integer <span>optional</span>
+						</dd>
+						<dd>Provides ability to restrict list of groups returned based on the office there associated with.</dd>
+						<dt>underHierarchy</dt>
+						<dd>
+							String <span>optional</span>
+						</dd>
+						<dd>Use the office hierarchy string to return all groups under a given hierarchy.</dd>
+
+						<dt>name</dt>
+						<dd>
+							String <span>optional</span>
+						</dd>
+						<dd>Use name of groups to restrict results.</dd>
+
+						<dt>externalId</dt>
+						<dd>
+							String <span>optional</span>
+						</dd>
+						<dd>Use externalId of groups to restrict results.</dd>
+
+						<dt>sqlSearch</dt>
+						<dd>
+							String <span>optional</span>
+						</dd>
+						<dd>Use an sql fragment valid for the underlying group schema to filter results. e.g. display_name like %K%</dd>
+						<dt>orphansOnly</dt>
+						<dd>
+							Boolean <span>optional</span>, defaults to false
+						</dd>
+						<dd>Use orphansOnly as true to list groups which are not associated to any center/parent.</dd>
+					</dl>
+					<p>Example Requests:</p>
+					<div class=apiClick>groups</div>
+					<br>
+					<div class=apiClick>groups?fields=name,officeName,joinedDate</div>
+					<br>
+					<div class=apiClick>groups?offset=10&limit=50</div>
+					<br>
+					<div class=apiClick>groups?orderBy=name&sortOrder=DESC</div>
+				</div>
+				<div class="method-example">
+					<code class="method-declaration">
+GET https://DomainName/fineract-provider/api/v1/groups?paged=true
+					</code>
+					<code class="method-response">
+{
+  "totalFilteredRecords": 2,
+  "pageItems": [
+    {
+      "id": 4,
+      "name": "AnotherGroup",
+      "status": {
+        "id": 100,
+        "code": "clientStatusType.pending",
+        "value": "Pending"
+      },
+      "active": false,
+      "officeId": 1,
+      "officeName": "Head Office",
+      "hierarchy": ".4."
+    }
+  ]
+}
+					</code>
+				</div>
+			</div>
+			<!-- end of group api docs -->
+
+            <!--Getting GSIM: -->
+            <a id="gsim_list" name="gsim_list" class="old-syle-anchor">&nbsp;</a>
+            <div class="method-section">
+                <div class="method-description">
+                    <h4>Retrieve a GSIM Application</h4>
+                    <p>Associations: String optional
+                        One of, or comma separated of GSIM Applications or all for all associations
+                    </p>
+
+                </div>
+                <div class="method-example">
+                    <code class="method-declaration">
+GET  https://DomainName/fineract-provider/api/v1/groups/{groupId}/gsimaccounts
+                    </code>
+
+
+                    <code class="method-response">
+[{
```

Comment:
check the formatting of other apis and align it accordingly. 
you may also use this tool for formatting the JSON body:
https://jsonformatter.curiousconcept.com/